.gitignore

```diff
@@ -0,0 +1,4 @@
+test.md
+node_modules
+package-lock.json
+package.json
```

Comment:
- [ ] **[OPTIONAL]:** Great job ignoring both `test.md` and `node_modules` as required! 💯 However, for future projects, avoid ignoring `package.json` and `package-lock.json`. Since these files store an exact reference to all your project's dependencies and their respective versions, it is important to keep those on your version control system, especially for collaborative work, where the team should have a consistent development environment. 🔧 🤓